clients/drcachesim/reader/config_reader.cpp

```diff
@@ -145,6 +145,18 @@ config_reader_t::configure(std::istream *config_file, cache_simulator_knobs_t &k
             } else {
                 knobs.model_coherence = false;
             }
+        } else if (param == "use_physical") {
```

Comment:
It would also be nice to have a test: should just take 2 lines to get a simple sanity check in one of the existing tests: add the param to tests/single_core.conf and add a knobs check in tests/config_reader_test.cpp.

Reply:
Thanks for pointing me to the right places, it makes this much easier. It looks like that test is never run. I can't find any mention of that cpp file in any of the CMakeLists.txt files.

Reply:
I don't think I was. It would've failed if it had been running.

Reply:
Hmm, I think you are right: commit fa4cc4a651d34e1afc8e29e8740a18563505c205 added the file but seems to have failed to either build it or make it a test!  Wow.   These are the real tests with "config" for x86-64:
```
  Test #285: code_api|tool.drcachesim.simple-config-file
  Test #288: code_api|tool.drcachesim.missfile-config-file
  Test #305: code_api|tool.drcachesim.threads-with-config-file
```
Do you feel up to adding it to drcachesim/CMakeLists.txt?  Model it on tool.drcachesim.unit_tests add_executable..add_test.